### PR TITLE
Minor cleanup for Flex SDK

### PR DESCRIPTION
**Pull request by arcnmx**
Trying to resolve some misc issues that get in the way when building with the [Apache Flex](https://flex.apache.org/) AS3 compiler (as well as on case-sensitive filesystems).

### CODE REVIEW

File: src/classes/mp/views/MPRoomViewFFR.as [hunk 1]
```diff
@@ -704,7 +704,7 @@ internal class UserPanel extends Sprite
     private var view:MPRoomViewFFR;
     private var room:MPRoomFFR;
 
-    private var name:Text;
+    private var _name:Text;
```
Comment (mat100payette) [inline src/classes/mp/views/MPRoomViewFFR.as:707]: That compiler doesn't support `name` as a var identifier? Haha

Comment (arcnmx) [reply src/classes/mp/views/MPRoomViewFFR.as:707]: nah, it's specifically upset that `name` already exists as a property on its superclass (Sprite -> DisplayObject looks like?)

Comment (mat100payette) [reply src/classes/mp/views/MPRoomViewFFR.as:707]: Ahhh okok. Well may I suggest using something like `panelName` instead? Using a leading underscore for this variable only might be misleading. Nitpick.

**Review by mat100payette — APPROVED** [review]

File: src/classes/mp/views/MPRoomViewFFR.as (hunk 1 above)
Comment (arcnmx) [reply src/classes/mp/views/MPRoomViewFFR.as]: mm it is a bit inconsistent... [pushed](https://github.com/flashflashrevolution/rCubed/compare/99251ad728b0d513b9f933fd4c793234f431e2c7..6af39c22cb2c5fe5978cc6c036f0d56017a08ac9)

**Review by Zageron — APPROVED** [review]

**Pull request merged**